workflows/methylation/methylation-cohort.wdl

```diff
@@ -199,7 +199,7 @@ task generate_umap {
     Int disk_size_gb = ceil(size(filtered_beta_values, "GiB") * 2) + 2
 
     command <<<
-        python $(which generate_umap.py) \
+        python /usr/local/bin/generate_umap.py \
```

Comment:
Shouldn't the script be on `$PATH`? Can we avoid using the full path? (Asking purely for aesthetic reasons, not a big deal at all if the full path is needed).

Reply:
`/usr/local/bin` should be on `$PATH`. `generate_umap.py` is executable but lacks a shebang line, so it can't actually be called directly currently. Calling with `python` doesn't follow `$PATH` resolution.

Reply:
Personally, I prefer using full paths for fixed location items in Docker containers (as evidenced by this command).

Reply:
Ah gotchya. Ok. Fine with that. Go ahead and run `update_container_values.sh` and I'll approve.